aten/src/ATen/native/cuda/MiscUtils.h

```diff
@@ -81,7 +81,6 @@ static inline Storage pin_memory(int64_t size) {
   int64_t adjusted_size = size * sizeof(T);
   return Storage(
       Storage::use_byte_size_t(),
-      caffe2::TypeMeta::Make<uint8_t>(),
```

Comment:
Need to check that downstream users get `uint8_t` set correctly.

Reply:
The only call site of this `pin_memory` overload is here: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cuda/BatchLinearAlgebra.cu#L528

It doesn't depend on `uint8_t` since it casts the storage data to an explicit pointer type.